frontend/packages/ceph-storage-plugin/console-extensions.json

```
@@ -79,6 +79,24 @@
       "required": ["MCG"]
     }
   },
+  /* Activates when RGW is present without MCG */
+  {
+    "type": "console.navigation/resource-cluster",
+    "properties": {
+      "id": "objectbuckets",
+      "section": "storage",
+      "name": "%ceph-storage-plugin~Object Buckets%",
+      "model": {
+        "group": "objectbucket.io",
+        "kind": "ObjectBucket",
+        "version": "v1alpha1"
+      }
+    },
+    "flags": {
+      "required": ["RGW"],
+      "disallowed": ["MCG"]
```

cant we do following instead of adding in a new extension ?

```suggestion
    "flags": {
      "required": ["RGW", "MCG"],
```

---

Required flags use a `AND` logic. So to create an `OR` logic we need to a new extension.